packages/mds-schema-validators/validators.ts

```diff
@@ -418,4 +455,67 @@ export function rawValidatePolicy(policy: Policy): Joi.ValidationResult<Policy>
   return Joi.validate(policy, policySchema)
 }
 
+export function validateDevice(device: Device) {
+  const { error } = Joi.validate(device, deviceSchema)
+  return error
+}
+
+const validateTripEvent = (event: VehicleEvent) => {
+  const { error } = Joi.validate(event, tripEventSchema)
+  return error
+}
+
+const validateProviderPickUpEvent = (event: VehicleEvent) => {
+  const { error } = Joi.validate(event, providerPickUpEventSchema)
```

Comment:
Curious why these aren't run through the internal `Validate` function that calls `Joi.validate` ... would want to use the `assert: false` options to avoid throwing exceptions.

Reply:
They are now :) I ended up going with keeping `assert: true` so that when we do eventually want these to bubble up as errors in agency we can just remove the try/catch.